packages/block-library/src/gallery/transforms.js

```diff
@@ -235,6 +225,31 @@ const transforms = {
 					},
 				},
 			},
+			__experimentalTransform( { named: { ids, columns = 3, link } } ) {
```

Comment:
I don't think the [`shortcode`](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-transforms/#shortcode) type has a `transform` method? Or is there something else I'm missing 😅?

Reply:
Yep, shortcode transforms don't currently support the `transform` method, so this would be introducing parity with block, file and other transforms. I think it's a good idea.

I think it'd be ok to ship it as stable, btw. The only part I'm unsure about is what `named:` means as I haven't done a shortcode transform before. What other keys can be in that object?

API seems to be pretty much what's expected, so I think it's ok to ship it and document it.

Reply:
I think `named:` is used to indicate that it is a `named` shortcode attribute and not a `numeric` shortcode attribute.

Reply:
No, you're right. I forgot that each type had its schema.

I agree with @talldan that it's ok to ship this as stable.

Reply:
Thanks for the feedback! I've updated the code to stabilize `transform`. Unfortunately, it doesn't play nice with the existing `attributes` field, and some of the features in `attributes` can't easily be done in `transform` (selectors).

I decided to only allow one at a time for now, but let me know if anyone has concerns or better ideas!